app/react/kubernetes/queries/useEvents.ts

```diff
@@ -27,22 +32,15 @@ async function getNamespaceEvents(
   }
 }
 
-export function useNamespaceEventsQuery(
+export function useEvents(
   environmentId: EnvironmentId,
-  namespace: string,
+  namespace?: string,
   options?: { autoRefreshRate?: number },
-  labelSelector?: string
+  params?: Params
```

Comment:
options should always be last.
but IMO the best it to put everything together:
```ts
export function useEvents(
  environmentId: EnvironmentId,
  {}: {
    options?: { autoRefreshRate?: number };
    params?: { namespace } & Params;
  } = {}
) {
	...
}
```

Reply:
updated